### DISCUSSION TITLE
Media Queries

### DISCUSSION THREAD

**Discussion by avedetvedea** [nativewind/nativewind#402] [Q&A] [answered]
Hey, I hesitated to ask because I'm not sure whether its lack of my understanding, a TW thing or related to NW. However, I figured I get some help regardless, because I'm stuck with mobile-breakpoint'ed media queries.

With the following custom setup:
```
screens: {
      xs: { raw: '(min-height: 568px)' },
      sm: { raw: '(min-height: 667px)' }, 
      md: { raw: '(min-height: 736px)' },
      lg: { raw: '(min-height: 812px)' }, 
      xl: { raw: '(min-height: 896px)' },
    },
```

I get unexpected results. For example, with 2 iPhones, an 8 (667px) and a 13 Max (926px),  a `Text` component having 
`text-3xl xs:text-9xl` makes both 3xl, 
`text-3xl sm:text-9xl` makes both 9xl,
`text-3xl md:text-9xl` makes bigger phone 9xl smaller 3xl, 
`text-3xl lg:text-9xl` makes bigger phone 3xl smaller 9xl, 
`text-3xl xl:text-9xl` makes both 9xl, 

which I couldn't debug. My original idea was to have `min-height` and `min-width` connected with `and` to aim  iPhones correctly, but failed with similar unexpected patterns.

**Comment by avedetvedea** [comment, marked as answer]
I tried changing the order but it seems I did something wrong, it works with reverse ordering, i.e., higher to lower for correct overriding.

**Reply by dannyhw** [reply]
Hey I'm actually having a lot of trouble with this but I don't really understand what you mean by "higher to lower for correct overriding" for me neither seems to work properly 

for example `h-md:mt-10 h-xs:mt-2` always uses mt-2 and `h-md:mt-10 mt-2` also always uses mt-2

what ended up working for you?

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
